test/doc/math/glaisher.cpp

```diff
@@ -0,0 +1,39 @@
+#include <eve/module/math.hpp>
+#include <eve/wide.hpp>
+#include <iostream>
+#include <iomanip>
+
+using wide_ft = eve::wide<float>;
+using wide_dt = eve::wide<double>;
+
+int main()
+{
+  wide_ft wxf;
+  wide_dt wxd;
+
+  std::cout << "---- simd"  << std::setprecision(9) << std::endl
```

Comment:
Those doc tests are rather larger for the info they need to provide. Do we need that much in the doc test ?
the fact we can do as(x) or as<T>() is not something we need to put **everywhere**, the as()  doc is there for that.